docs/docs/docs/getting-started/installation.md

```
@@ -255,6 +255,23 @@ These steps are specific to Linux. You will need to modify them accordingly for
 
 All done!
 
+#### CLI Shutdown (Development)
+
+Use the command `docker compose` command to cleanly shutdown the dev container
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)